load_default_pseudomonas_data.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Script to load default Pseudomonas data (in the files included in this
+# repository) into the adage-server database via management commands.
+
+# Import Pseudomonas annotation data
+python manage.py import_data ../data/PseudomonasAnnotation.tsv
+
+# Create Pseudomonas object in database for django-organisms
+python manage.py organisms_create_or_update \
+    --taxonomy_id=208964 \
+    --scientific_name="Pseudomonas aeruginosa" \
+    --common_name="Pseudomonas aeruginosa"
+
+# Add new machine learning model into database
+python manage.py add_ml_model "Ensemble ADAGE 300" 208964 --g2g_edge_cutoff 0.4
+
+# Import activity data into the database:
+python manage.py import_activity \
+    ../data/all-pseudomonas-gene-normalized_HWActivity_perGene_with_net300_100models_k=300_seed=123_ClusterByweighted_avgweight_network_ADAGE.txt \
+    "Ensemble ADAGE 300"
+
+# Save Ensembl and PseudoCAP Cross-reference Databases
+python manage.py genes_add_xrdb \
+    --name=Ensembl \
+    --URL=http://www.ensembl.org/Gene/Summary?g=_REPL_
+
+python manage.py genes_add_xrdb \
+    --name=PseudoCAP \
+    --URL=http://www.pseudomonas.com/getAnnotation.do?locusID=_REPL_
+
+# Retrieve data file for PAO1 genes and load it:
+wget -qO - "ftp://ftp.ncbi.nih.gov/gene/DATA/GENE_INFO/Archaea_Bacteria/Pseudomonas_aeruginosa_PAO1.gene_info.gz" \
+    | zcat > ../data/Pseudomonas_aeruginosa_PAO1.gene_info
+
+# Call genes_load_geneinfo to populate the database:
+python manage.py genes_load_geneinfo \
+    --geneinfo_file=../data/Pseudomonas_aeruginosa_PAO1.gene_info \
+    --taxonomy_id=208964 \
+    --systematic_col=3 \
+    --symbol_col=2 \
+    --put_systematic_in_xrdb=PseudoCAP
+
+wget -qO - "ftp://ftp.ncbi.nih.gov/gene/DATA/gene_history.gz" \
+    | zcat > ../data/gene_history
+
+python manage.py genes_load_gene_history ../data/gene_history 208964 \
+    --tax_id_col=1 \
+    --discontinued_id_col=3 \
+    --discontinued_symbol_col=4
+
+# Import gene-gene network data into the database:
+python manage.py import_gene_network \
+    ../data/eADAGE_net300_allNodes_ADAGEnet_PAID_corCutoff0.4.txt \
+    "Ensemble ADAGE 300"
+
+# Note that the ParticipationType "High-weight genes" has been
+# created in: migrations/0009_auto_20170503_1700.py
+# Import node-gene participation data into the database:
+python manage.py import_node_gene_network ../data/node_gene_network.txt \
+    "Ensemble ADAGE 300" \
+    "High-weight genes"
+
+python manage.py import_gene_sample_expr \
+    ../data/all-pseudomonas-gene-normalized.pcl \
+    208964
+
+python manage.py tribe_client_pickle_public_genesets
+
+# Build haystack search indexes:
+python manage.py update_index
```

Comment:
The way we've been doing this via the `fab` scripts is a full rebuild command, which looks like:

```bash
python manage.py rebuild_index --noinput
```

I think that `update_index` will do mostly the same thing if we're starting with fresh data, as we are here, but have you tested? Also: does `update_index` have a `--noinput` parameter akin to `rebuild_index`? I feel like we needed that but maybe it was just to get past the confirmation about clobbering old indexes, which isn't relevant for `update_index`, I guess.

Reply:
Ah, ok, I will look into this and see which one we want.

Reply:
I think the long-term strategy is to integrate with a `data-refinery` to help harvest new data, so, ultimately, we want an `update_index` to run automatically after new data are imported.

But right now we only import data to initialize the database. For now, I just want to make sure what we have works.

Reply:
That's right - I think the `update_index` works, and as you said, it does not need (or even have) the `--noinput` parameter since no indexes would be clobbered. I don't have strong feelings either way. Do you?